android/clientlib-ktx/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
Nitpick: the 2 proguard files are probably not needed?

Reply:
Good question 🤔 Usually proguarding libraries (especially open-source) is not advised since it would make it more difficult to debug/reverse for developers importing it. I guess we could just run the "optimization" part of proguard, that's probably not a priority?

Reply:
I'll just remove for now; if we need it we can re-add.